tests/execution_engine/test_pandas_execution_engine.py

```diff
@@ -27,7 +35,138 @@
 from tests.expectations.test_util import get_table_columns_metric
 
 
-def test_constructor():
+@pytest.fixture(scope="function")
+def aws_credentials():
+    """Mocked AWS Credentials for moto."""
+    os.environ["AWS_ACCESS_KEY_ID"] = "testing"
+    os.environ["AWS_SECRET_ACCESS_KEY"] = "testing"
+    os.environ["AWS_SECURITY_TOKEN"] = "testing"
+    os.environ["AWS_SESSION_TOKEN"] = "testing"
+
+
+@pytest.fixture
+def s3(aws_credentials):
+    with mock_s3():
+        yield boto3.client("s3", region_name="us-east-1")
+
```

Comment:
Moved fixtures to the top. If you think it makes more sense to have fixtures close to where they are used, I'm happy to do that as well!

Reply:
❤️